sound/soc/sof/Kconfig

```diff
@@ -79,6 +79,20 @@ config SND_SOC_SOF_NOCODEC
 	  Say Y if you need this nocodec fallback option
 	  If unsure select "N".
 
+config SND_SOC_SOF_STRICT_ABI_CHECKS
+	bool "SOF strict ABI checks"
+	help
+	  This option enables strict ABI checks for firmware and topology
+	  files.
+	  When these files are more recent than the kernel, the kernel
+	  will handle the functionality it supports and may report errors
+	  during topology creation or run-time usage if new functionality
+	  is invoked.
+	  This option will stop topology creation and firmware load upfront.
+	  It is intended for SOF CI/releases and not for users or distros.
+	  Say Y if you want strict ABI checks for an SOF release
```

Comment:
@plbossart looks good. just a couple of nits.
"a SOF release" instead of an

Reply:
ok

Reply:
actually 'an' is easier to say. not sure what the proper grammar is here...

Reply:
@plbossart definitely "a" is the right grammar

Reply:
it's 'an'. sending you an SOS...You use 'an' before vowel sounds, even if they start with a consonant.

Reply:
but how does SOF start with a consonant?

Reply:
@ranj063 

> It's "an," because you pronounce is "Ess-Oh-Ess," with a vowel sound, and the sound is the important thing.

https://answers.yahoo.com/question/index?qid=20110805214715AAbMWkZ&guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAABe-bbuq_hIKk8F7_mZgTFqlXfvAKJSyQuz3_QIg4eSccT2LzwC-mM5nvp67vcQ0JvdfM8e5PnYAN-dirprnJps9hLoKEW7nDeCT0YIu-3Y6MQxX3_ROh4J35YYZsLc0eP1ZvzRK7zHUSAkgT_jec2SIIwRuiA9Rz45Di_iXnQV7

Reply:
@xiulipan agree thats true if you spell the letters in SOF as in  S.O.F. It reads bad when I read as "SOF" as a word. anyway this is silly. lets move on.